tools/eubfr-cli/bin/cli.js

```diff
@@ -79,42 +77,39 @@ program.command('demo', 'Demo applications management');
  *
  * ### Notes
  *
- * If you want to make use of the CLI to automatically upload or delete all content of a given stage, you can optionally create a `.content` folder in the root of your project, with the following example structure:
+ * The [eubfr-content](https://s3.console.aws.amazon.com/s3/buckets/eubfr-content/?region=eu-central-1&tab=overview) S3 bucket is a central content repository which holds files which could be ingested by the data lake.
```

Comment:
FYI, I jumped into this section and ran `yarn content:download` without setting `EUBFR_CONTENT_REPOSITORY`, 'cause it wasn't obvious at this point. Just saying :)